public/app/plugins/datasource/cloudwatch/datasource.js

```diff
@@ -332,6 +333,36 @@ function (angular, _, moment, dateMath, CloudWatchAnnotationQuery) {
       });
     }
 
+    this.expandTemplateVariable = function(targets, templateSrv) {
```

Comment:
Would be nice to splitt this functions into smaller functions. Makes it easier to read the code.